bundles/net.tourbook/src/net/tourbook/ui/views/tourDataEditor/TourDataEditorView.java

```diff
@@ -4423,16 +4427,15 @@ private void createUI_Section_144_Weather_Temperature_Device_Col2(final Composit
             _txtWeather_Temperature_Average_Device = _tk.createText(
                   container,
                   UI.EMPTY_STRING,
-                  SWT.TRAIL);
+                  SWT.READ_ONLY);
```

Comment:
For the user it would be helpful to know the source of the avg/min/max temperature values, can you update/replace the tooltips?

Reply:
You have had created tooltips but placed them at the wrong location. The device value controls had no tooltips, why I couldn't see it, I'll fix it

Tooltip OK

![image](https://user-images.githubusercontent.com/1283445/158525291-3e49863b-92ab-4a04-a4d3-b46face409bd.png)


Tooltip not OK

![image](https://user-images.githubusercontent.com/1283445/158525328-68ccf856-383d-4167-9b16-3f618adf2ebb.png)

Reply:
I understand now, a screenshot/picture is worth a 1000 words :-)